app/models/parsers/cardinal_pbmc_count_parser.rb

```diff
@@ -44,11 +44,18 @@ def qc_data
       @qc_data ||=
         {}.tap do |qc_data|
           rows.each do |row|
-            qc_data[row[0]] = { viability: Unit.new(row[4]), live_cell_count: Unit.new(row[2], 'cells') }
+            qc_data[get_well_location(row[0])] = {
+              viability: Unit.new(row[4]),
+              live_cell_count: Unit.new(row[2], 'cells')
+            }
           end
         end
     end
 
+    def get_well_location(cell)
+      cell.try(:split, ':')[1]
```

Comment:
Why is .try() needed? If the string does not have a ':' separator, don't we want it to fail?

Reply:
good point